dietpi/dietpi-software

```diff
@@ -8966,6 +9010,43 @@ _EOF_
 			pihole -w freegeoip.app --comment 'Used by dietpi-banner and dietpi-vpn to obtain your public IP and location'
 
 		fi
+		
+		software_id=171 # AdGuardHome
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+			
+			if [[ ! -f '/opt/AdGuardHome/AdGuardHome.yaml' ]]; then
+				dps_index=$software_id Download_Install 'AdGuardHome.yaml' /opt/AdGuardHome/AdGuardHome.yaml
+				G_CONFIG_INJECT 'password:[[:blank:]]' "  password: $(htpasswd -bnBC 10 "" $GLOBAL_PW | tr -d ':\n' | sed 's/$2y/$2a/')" /opt/AdGuardHome/AdGuardHome.yaml
+			fi
+
+			# Service
+			cat << _EOF_ > /etc/systemd/system/AdGuardHome.service
+[Unit]
+Description=AdGuard Home: Network-level blocker
+ConditionFileIsExecutable=/opt/AdGuardHome/AdGuardHome
+After=syslog.target network-online.target
+
+[Service]
+StartLimitInterval=5
+StartLimitBurst=10
+ExecStart=/opt/AdGuardHome/AdGuardHome "-s" "run"
+
+WorkingDirectory=/opt/AdGuardHome
+
+StandardOutput=file:/var/log/AdGuardHome.out
+StandardError=file:/var/log/AdGuardHome.err
+Restart=always
+RestartSec=10
+EnvironmentFile=-/etc/sysconfig/AdGuardHome
```

Comment:
`/etc/sysconfig` is more a CentOS-ish standard, while on Debian-based distros `/etc/default` is used instead.

Reply:
ok I changed it but it doesn't seems to be used

Reply:
I also looked into the [docs](https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration) but it doesn't seem to take care of environment variables. In this case, I'd just remove that part. We could do something like:
```
EnvironmentFile=-/etc/default/adguardhome
ExecStart=/opt/AdGuardHome/AdGuardHome -s start $CMD_ARGS
```
and then allow users to set `CMD_ARGS="something"` in `/etc/default/adguardhome`. But from what I see, all important settings can be set via the YAML configuration file already. If someone really wants to e.g. disable update checks, `dietpi-services` can be used to create a service override config.

Reply:
ok I removed the `EnvironmentFile` for now